### PR TITLE
Update secret-scanning partner onboarding to add more 'gotcha' information for signature validation

### PR DESCRIPTION
**Pull request by just-joshing**
### Why:

We received partner feedback that they encountered difficulty implementing signature validation because it wasn't clear that the raw payload needed to be used.

They use IBM Cloud Functions Actions which by default parse JSON request payloads.
So when they would stringify the JSON again and find signature validation was failing, it wasn't clear that the issue was stringifying the JSON again rearranged key/value members and changed spacing.

### What's being changed:

This change updates:
- the sample JSON payloads to have no spaces, like production
- the URL for getting the public key in the samples which was pointing to the old endpoint `token_scanning`
- the first token sample to only include a token, not `X-Header-Banner` as well which wouldn't be included in a payload

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

### CODE REVIEW

**Comment by just-joshing** [comment]
cc @github/dsp-token-scanning-reviewers

**Review by greysteil — DISMISSED** [review]

**Review by aashah — DISMISSED** [review]
neat

File: content/developers/overview/secret-scanning.md [hunk 1]
```diff
@@ -90,12 +84,18 @@ You can retrieve the {% data variables.product.prodname_dotcom %} secret scannin
 
 {% note %}
 
-**Note**: When you send a request to the public key endpoint above, you may hit rate limits. To avoid hitting rate limits, you can use a personal access token as suggested below, or use a conditional request. For more information, see "[Getting started with the REST API](/rest/guides/getting-started-with-the-rest-api#conditional-requests)."
+**Note**: When you send a request to the public key endpoint above, you may hit rate limits. To avoid hitting rate limits, you can use a personal access token (no scopes required) as suggested in the samples below, or use a conditional request. For more information, see "[Getting started with the REST API](/rest/guides/getting-started-with-the-rest-api#conditional-requests)."
```
[the remark this answers is not in the document]
Comment (aashah) [reply content/developers/overview/secret-scanning.md:87]: Edit: they do have it...nevermind

[the remark this answers is not in the document]
Comment (just-joshing) [reply content/developers/overview/secret-scanning.md:87]: 👍

**Review by aashah — APPROVED** [review]
🤙

File: content/developers/overview/secret-scanning.md [hunk 2]
```diff
@@ -90,12 +84,18 @@ You can retrieve the {% data variables.product.prodname_dotcom %} secret scannin
 
 {% note %}
 
-**Note**: When you send a request to the public key endpoint above, you may hit rate limits. To avoid hitting rate limits, you can use a personal access token as suggested below, or use a conditional request. For more information, see "[Getting started with the REST API](/rest/guides/getting-started-with-the-rest-api#conditional-requests)."
```
Comment (QbZn) [inline content/developers/overview/secret-scanning.md:93]: b******0@gmail.com

**Review by QbZn — CHANGES_REQUESTED** [review]

**Comment by janiceilene** [comment]
@just-joshing Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Review by greysteil — APPROVED** [review]

**Review by skedwards88 — APPROVED** [review]
🎉 Thank you for opening this PR! I'll get these changes merged down for you.

**Pull request merged**